packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VisXZoom/VisXZoom.js

```diff
@@ -1,13 +1,16 @@
+import { useWheel } from '@use-gesture/react'
 import { localPoint } from '@visx/event'
 import { Zoom } from '@visx/zoom'
-import { throttle } from 'lodash'
+import throttle from 'lodash/throttle'
 import { observer } from 'mobx-react'
 import PropTypes from 'prop-types'
 import { useEffect, useRef } from 'react'
 
 import { useKeyZoom } from '@hooks'
 import ZoomEventLayer from '../ZoomEventLayer'
 
+export const ZOOM_HOT_KEY = 'ctrlKey'
```

Comment:
This change is so it's easier to tinker with which key to use as we discuss the various key options (shiftKey, ctrlKey, metaKey, altKey, other?).

Reply:
If you choose `metaKey`, be aware that it doesn't work in Firefox < 188, and one of the NfN moderators is stuck on FF 115 (also, don't make decisions like this without including user research!)

https://github.com/zooniverse/front-end-monorepo/issues/6592

Reply:
Ctrl + tap brings up the context menu with a trackpad, so Ctrl might not have been a great choice on my part. That said, there is a clear difference between single finger tap to click and two finger drag to scroll, and the latter won't be expected to open a context menu.

Reply:
Added the info regarding the meta key and control key, as well as a few other notes about the potential hot keys in https://github.com/zooniverse/front-end-monorepo/issues/6758#issuecomment-2730967783 .